src/user/users-storage.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@nestjs/common';
+import { UpdateUserDto } from './dto/update-user.dto';
+import { User } from './entities/user.entity';
+
+@Injectable()
+export class UsersDataBase {
```

Comment:
I think it should be service, because it looks and behaviors absolutely like service

@Optional
Do we really need this file? May be we could implement this logic in UserService

Reply:
Of course, we can store the database in the service, but this is not correct in terms of architecture. The service should have only business logic